justfile

```diff
@@ -13,22 +13,27 @@ _gotools:
   go fmt ./...
   go vet {{go_flags}} ./...
 
+_rusttools:
+  cargo fmt
+  cargo clippy --features bolero
+
 # Run linters against code (incl. license headers)
-lint: _license_headers _gotools
+lint: _license_headers _gotools _rusttools
 
 _path := `echo $PATH`
-gen: _protoc _protoc_gen_go _protoc_gen_go_grpc && lint
+gen: _protoc _protoc_gen_go _protoc_gen_go_grpc
```

Comment:
CI is already running lint after gen so I see no problem doing it

Reply:
Does the && stuff run after the rule executes then?

Reply:
I don't know what was originally wrong, but these changes are clearly not needed, I've reverted the commit.